tests/test_freshness_date_parser.py

```diff
@@ -57,6 +57,21 @@ def test_relative_past_dates_with_time_as_period(self, date_string, ago, period)
 
     @parameterized.expand([
         # English dates
+        param('1 fortnight', ago={'days': 14}, period='day'),
+        param('last fortnight', ago={'days': 14}, period='day'),
+        param('14 fortnight', ago={'days': 196}, period='day'),
+        param('a fortnight ago', ago={'days': 14}, period='day'),
+        param('last fortnight', ago={'days': 14}, period='day'),
```

Comment:
I believe this is repeated and the same as in line 61.

Reply:
@gutsytechster Done these changes, Please Look into it 😊.